mne/viz/_3d.py

```diff
@@ -866,7 +868,8 @@ def plot_alignment(info=None, trans=None, subject=None, subjects_dir=None,
     skull_alpha = dict()
     skull_colors = dict()
     hemi_val = 0.5
-    max_alpha = 1.0 if len(other_picks['seeg']) == 0 else 0.75
+    max_alpha = 1.0 if len(other_picks['seeg']) == 0 or \
+        len(other_picks['dbs']) == 0 else 0.75
```

Comment:
I think you need `and` rather than `or` here and we might as well reformulate / DRY (don't repeat yourself) the code a bit
```suggestion
    no_deep = all(len(other_picks[key]) == 0 for key in ('dbs', 'seeg'))
    max_alpha = 1.0 if no_deep else 0.75
```